src/lib/Preview.js

```diff
@@ -1239,7 +1239,6 @@ class Preview extends EventEmitter {
                 this.ui.finishProgressBar();
                 break;
             case VIEWER_EVENT.mediaEndAutoplay:
```

Comment:
This entire condition can be removed, as it's now a no-op.

Reply:
ok

Reply:
Actually, it looks like it's preventing the event from being forwarded by the `default` condition.

Reply:
Then the test for it should be removed entirely too, right?

Reply:
or change it to test the default condition